langchain4j-alloydb-pg/src/main/java/dev/langchain4j/data/document/loader/alloydb/AlloyDBLoader.java

```diff
@@ -35,95 +35,140 @@ public class AlloyDBLoader {
     private final BiFunction<Map<String, Object>, List<String>, String> formatter;
     private final String metadataJsonColumn;
     private final Connection connection;
-    private static final String DEFAULT_CONTENT_COL = "page_content";
-    private static final String DEFAULT_METADATA_COL = "langchain_metadata";
     private static final Logger log = LoggerFactory.getLogger(AlloyDBEngine.class.getName());
+    private static final String DEFAULT_METADATA_COL = "langchain_metadata";
 
-    private AlloyDBLoader(Connection connection, String query, List<String> contentColumns, List<String> metadataColumns,
-            BiFunction<Map<String, Object>, List<String>, String> formatter, String metadataJsonColumn) {
+    private AlloyDBLoader(Connection connection, String query,
```

Comment:
This should accept the builder and use the builder attributes to set the loader variables